### PR TITLE
Updated Announcement And User Controller With new Roles

### PR DESCRIPTION
**Pull request by reyimanuel**
# **Updating announcement and user controller with the new roles**

Adding a new function that takes id from login token on userController and AnnouncementController

## **Deskripsi**

On the UserController, this system will show all the users with the role 'member', and only the users with role 'committe && coordinator' have the privilage to use this method. So as the announcement, only the user with role 'committe && coordinator' have the privilage to use the create, update, and delete method.

**ANNOUNCEMENT CRUD SYSTEM**
Endpoint Create:
image

Endpoint Update:
image

Endpoint Delete:
image

Endpoint as member:
image

**USER GET SYSTEM**
Endpoint as commite && coordinator:
image

Endpoint as member:
image

@zharkaa

### CODE REVIEW

File: database/seeders/GallerySeeder.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ public function run()
     {
         $eligibleUserIds = DB::table('users')
             ->join('roles', 'users.role_id', '=', 'roles.id')
-            ->whereIn('roles.name', ['coordinator', 'committee'])
+            ->whereIn('roles.name', ['Pengurus'])
```
Comment (zharkaa) [inline database/seeders/GallerySeeder.php:19]: yang ini harusnya tetap `coordinator` dan `comittee`.

apakah ada alasan kenapa sampe diganti ?

Comment (reyimanuel) [reply database/seeders/GallerySeeder.php:19]: OHH itu ada bantu adel pnya sih kak, jadi tcoba buat pqt smntra taso lupa ubah ulang

Comment (reyimanuel) [reply database/seeders/GallerySeeder.php]: oke so normal ulang kak

Comment (zharkaa) [reply database/seeders/GallerySeeder.php]: okee sip

**Review by zharkaa — APPROVED** [review]
LGTM!

**Review by zharkaa — APPROVED** [review]
LGTM!

**Pull request merged**